borsh/src/lib.rs

```diff
@@ -81,10 +81,12 @@ pub mod ser;
 
 pub use de::BorshDeserialize;
 pub use de::{from_reader, from_slice};
+
```

Comment:
these blanks change is so that release-plz picks up the commit for changelog, as the changes are otherwise outside of `borsh` or `borsh-derive` crates

Reply:
I think we don't necessarily need it as a line on the CHANGELOG, but rather we need a very prominent comment with a reference to the guide in the release notes to the release.